pkg/kubectl/apps/kind_visitor_test.go

```diff
@@ -0,0 +1,190 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apps_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"k8s.io/kubernetes/pkg/kubectl/apps"
+)
+
+var _ = Describe("When KindVisitor accepts a GroupKind", func() {
+
+	var visitor *TestKindVisitor
+
+	BeforeEach(func() {
+		visitor = &TestKindVisitor{map[string]int{}}
+	})
+
+	It("should Visit DaemonSet iff the Kind is a DaemonSet", func() {
```

Comment:
`iff --> if`  almost all of those descriptions have typo, do we made it on purpose ?